lib/config/massage.ts

```diff
@@ -58,8 +58,8 @@ export function massageConfig(config: RenovateConfig): RenovateConfig {
       ][]) {
         if (updateTypes.includes(key)) {
           const newRule = clone(rule);
-          newRule.updateTypes = rule.updateTypes || [];
-          newRule.updateTypes.push(key);
+          newRule.matchUpdateTypes = rule.matchUpdateTypes || [];
+          newRule.matchUpdateTypes.push(key);
```

Comment:
When TS 4.0 is available we can reduce it to:
```suggestion
          (newRule.matchUpdateTypes ??= []).push(key);
```
 🙃 

https://devblogs.microsoft.com/typescript/announcing-typescript-4-0-beta/#short-circuiting-assignment-operators

Reply:
Exactly what I thought reading release notes :)